src/MarketplaceOrdering/MarketplaceOrdering/help/Get-AzMarketplaceTerms.md

```diff
@@ -38,7 +38,7 @@ RetrieveDatetime  : <RetrieveDatetime>
 ## PARAMETERS
 
 ### -DefaultProfile
-The credentials, account, tenant, and subscription used for communication with azure.
+The credentials, account, tenant, and subscription used for communication with Azure.
```

Comment:
Need an update to the MarketPlaceOrdering changelog to mark this module for update